src/accelerate/utils/modeling.py

```diff
@@ -616,7 +616,73 @@ def check_tied_parameters_on_same_device(tied_params, device_map):
             )
 
 
-def find_tied_parameters(model: nn.Module, **kwargs):
+def _get_named_modules(
```

Comment:
If we're documenting it in this way, we should make this public :)

Reply:
not sure you need it public :) its just copied form PyTorch code

Reply:
I removed `Example` for this docstring, not sure it is needed for a private method, and also doc-builder was raising an error